### PR TITLE
Eliminate Cloned and update examples to support stable rust.

### PR DESCRIPTION
**Pull request by bluss**
Eliminate Cloned to support stable rust.

This is a change to public API so it is technically a breaking change.

### CODE REVIEW

**Comment by bluss** [comment]
Oh nice, it builds on stable but every single example in the docs fails.

**Comment by Gankro** [comment]
(the alias was a polyfill for lazyness, so it's not a *real* break)

**Comment by Gankro** [comment]
Ahaha that's amazing. I forgot to audit the doc examples for stuff.

**Comment by Gankro** [comment]
r=me with docs fixed

**Comment by bluss** [comment]
Great, it passes on stable.

File: src/lib.rs [hunk 1]
```diff
@@ -343,10 +342,9 @@ impl<B: BitBlock> BitVec<B> {
     /// # Examples
     ///
     /// ```
-    /// # #![feature(collections)]
-    /// use std::collections::BitVec;
+    /// use bit_vec::BitVec;
     ///
-    /// let bv = BitVec::from_bytes(&[0b10100000, 0b00010010]);
+    /// let bv = BitVec::<u32>::from_bytes(&[0b10100000, 0b00010010]);
```
Comment (bluss) [inline src/lib.rs]: All examples had to be updated like this. We should probably default new and from_bytes to just a specific block size. Can be generalized later.

**Comment by Gankro** [comment]
Merging as a good step.

**Pull request merged**